From 25f084e4f59a1ec00394c07f5f37c36226746180 Mon Sep 17 00:00:00 2001
From: Tk-Glitch <ti3nou@gmail.com>
Date: Sun, 4 Apr 2021 16:48:21 +0200
Subject: We do not want start.exe to stay around when using Proton as it prevents graceful exits.
So let's strip its invincibility shield so Death can come and take what it has to.


diff --git a/programs/start/start.c b/programs/start/start.c
index 3efad69b090..a5e420f9158 100644
--- a/programs/start/start.c
+++ b/programs/start/start.c
@@ -463,7 +463,7 @@ int __cdecl wmain (int argc, WCHAR *argv[])
 		}
                 else if (is_option(argv[i], L"/exec")) {
 			creation_flags = 0;
-			sei.fMask = SEE_MASK_NOCLOSEPROCESS | SEE_MASK_NO_CONSOLE | SEE_MASK_FLAG_NO_UI;
+			sei.fMask = SEE_MASK_NO_CONSOLE | SEE_MASK_FLAG_NO_UI;
                         i++;
                         break;
 		}
From fcb37c9c123cd841debb17b6821493b8b2f83473 Mon Sep 17 00:00:00 2001
From: Alexandre Julliard <julliard@winehq.org>
Date: Mon, 29 Mar 2021 12:22:55 +0200
Subject: [PATCH] wine.inf: Always install start.exe in system32.

Wine-Bug: https://bugs.winehq.org/show_bug.cgi?id=50867
Signed-off-by: Alexandre Julliard <julliard@winehq.org>
---
 loader/wine.inf.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index 52c2297fb2f7..de39c6223af2 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -2674,6 +2674,7 @@ HKLM,%CurrentVersion%\Telephony\Country List\998,"SameAreaRule",,"G"
 11,,explorer.exe
 11,,iexplore.exe
 11,,notepad.exe
+11,,start.exe
 11,,winetest.exe,-
 ; skip .NET fake dlls in Wine Mono package
 11,,aspnet_regiis.exe,-
